### PR TITLE
Added About Mohit

### PR DESCRIPTION
**Pull request by nagpalm7**
I've added information about myself

### CODE REVIEW

File: submissions/nagpalm7/about.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Interests
+Ethical Hacking
+# Contact Info
+Email: [nagpalm7@gmail.com](mailto:nagpalm7@gmail.com)
+Facebook profile: [Mohit Nagpal](https://www.facebook.com/9996567340m)
+Github: [nagpalm7](https://github.com/nagpalm7)
+# Emoji
+My favorite emoji is :haha:
+
+
```
Comment (azharcaptain20) [inline submissions/nagpalm7/about.md]: Remove extra lines

**Review by azharcaptain20 — CHANGES_REQUESTED** [review]

**Comment by nagpalm7** [comment]
I've made the changes. Please review

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Write a good commit message and contact info should not be in same line.

**Comment by divyam3897** [comment]
closing as duplicate.

**Pull request closed without merging**